contracts/RelayHub.sol

```diff
@@ -15,12 +15,14 @@ contract RelayHub is RelayHubApi {
     uint constant minimum_relay_balance = 0.5 ether;  // XXX TBD - can't register/refresh below this amount.
     uint constant public gas_reserve = 99999; // XXX TBD - calculate how much reserve we actually need, to complete the post-call part of relay().
     uint constant public gas_overhead = 47586;  // the total gas overhead of relay(), before the first gasleft() and after the last gasleft(). Assume that relay has non-zero balance (costs 15'000 more otherwise).
+    uint accept_relayed_call_max_gas = 50000;
 
     mapping (address => uint) public nonces;    // Nonces of senders, since their ether address nonce may never change.
 
     enum State {UNKNOWN, STAKED, REGISTERED, REMOVED, PENALIZED}
     // status flags for TransactionRelayed() event
     enum RelayCallStatus {OK, CanRelayFailed, RelayedCallFailed, PostRelayedFailed}
+    enum CanRelayStatus {OK, WrongSignature, WrongNonce, AcceptRelayedCallUnkownError, AcceptRelayedCallReverted}
```

Comment:
These two  enums are public APIs (they are returned by `can_relay` and events) , and thus belong to the API (interface), and not the relayHub itself.

Reply:
Returning enum as external function is converted to int anyway, so not sure if we should move it to RelayHubApi.sol which also means to change the abi of can_relay